cub/cub/device/device_segmented_reduce.cuh

```diff
@@ -464,35 +467,17 @@ struct DeviceSegmentedReduce
   //!
   //! The code snippet below illustrates the min-reduction of a device vector of ``int`` data elements.
   //!
-  //! .. code-block:: c++
-  //!
-  //!    #include <cub/cub.cuh>
-  //!    // or equivalently <cub/device/device_radix_sort.cuh>
-  //!
-  //!    // Declare, allocate, and initialize device-accessible pointers
-  //!    // for input and output
-  //!    int num_segments;   // e.g., 3
-  //!    int *d_offsets;     // e.g., [0, 3, 3, 7]
-  //!    int *d_in;          // e.g., [8, 6, 7, 5, 3, 0, 9]
-  //!    int *d_out;         // e.g., [-, -, -]
-  //!    ...
-  //!
-  //!    // Determine temporary device storage requirements
-  //!    void     *d_temp_storage = NULL;
-  //!    size_t   temp_storage_bytes = 0;
-  //!    cub::DeviceSegmentedReduce::Min(
-  //!      d_temp_storage, temp_storage_bytes, d_in, d_out,
-  //!      num_segments, d_offsets, d_offsets + 1);
+  //! .. literalinclude:: ../../test/catch2_test_device_segmented_reduce_api.cu
+  //!     :language: c++
+  //!     :dedent:
+  //!     :start-after: example-begin segmented-reduce-custommin
+  //!     :end-before: example-end segmented-reduce-custommin
```

Comment:
important: this should've been in the `cub::DeviceSegmentedReduce::Reduce` docs instead of `cub::DeviceSegmentedReduce::Min`.

Reply:
excellent catch